robot/pmm/resources/HomePageObject.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+from cumulusci.robotframework.pageobjects import BasePage
+from cumulusci.robotframework.pageobjects import pageobject
+from pmm import pmm_lex_locators
+
+
+@pageobject("Home", "Homepage")
+class PMMHomePage(BasePage):
+    object_name = None
+
+    def _go_to_page(self, **kwargs):
+        """To go to Home page"""
+        url = self.cumulusci.org.lightning_base_url
+        url = "{}/lightning/page/home".format(url)
+        self.selenium.go_to(url)
+        self.salesforce.wait_until_loading_is_complete()
+        time.sleep(2)
```

Comment:
Is there a reason you're doing a sleep here? Would it not be possible to wait for something on the page to appear rather than using a hard-coded sleep?